styles/mynoise/catppuccin.user.less

```
@@ -0,0 +1,339 @@
+    @text: @catppuccin[@@flavor][@text];
+    @subtext1: @catppuccin[@@flavor][@subtext1];
+    @subtext0: @catppuccin[@@flavor][@subtext0];
+    @overlay2: @catppuccin[@@flavor][@overlay2];
+    @overlay1: @catppuccin[@@flavor][@overlay1];
+    @overlay0: @catppuccin[@@flavor][@overlay0];
+    @surface2: @catppuccin[@@flavor][@surface2];
+    @surface1: @catppuccin[@@flavor][@surface1];
+    @surface0: @catppuccin[@@flavor][@surface0];
+    @base: @catppuccin[@@flavor][@base];
+    @mantle: @catppuccin[@@flavor][@mantle];
+    @crust: @catppuccin[@@flavor][@crust];
+    @accent: @catppuccin[@@flavor][@@accentColor];
+
+    color-scheme: if(@flavor = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    /* Start styling your website here: */
+    background-color: @base;
+    body, h1, h2 {
+      color: @text;
+    }
+    a {
+      color: @peach;
+    }
+    a:hover {
+      /* NOTE: a:hover also governs .button hover styling */
+      background-color: @peach;
+      color: @base;
+    }
+    #topMenu {
+      background-color: @surface0;
+      border-color: @surface2;
+    }
+    .menuDivider {
+      border-bottom-color: @text;
+      color: @text;
+    }
+    .button {
+      background-color: @overlay0;
+      border-color: @overlay2;
+      color: @text;
+    }
+    .menutext {
+      color: @text;
+    }
+    #newbullet {
+      /* XXX: do not remove !important, this particular element is inline-styled */
+      color: @red !important;
+    }
+    .mainTitle {
+      color: @text;
+    }
+    .subTitle {
+      color: @subtext1;
+    }
+    .ui-slider-handle {
+      box-shadow: inset 0 0 0 4px @base;
+    }
+    #s0 .ui-slider-handle {
+      background: @rosewater;
+      /* XXX: do not refactor! an !important in the site's stylesheet won't let this declaration work with a lower specificity */
+      border-color: @overlay0 !important;
+    }
+    #s1 .ui-slider-handle {
+      background: @red;
+      border-color: @overlay0 !important;
+    }
+    #s2 .ui-slider-handle {
+      background: @peach;
+      border-color: @overlay0 !important;
+    }
+    #s3 .ui-slider-handle {
+      background: @yellow;
+      border-color: @overlay0 !important;
+    }
+    #s4 .ui-slider-handle {
+      background: @green;
+      border-color: @overlay0 !important;
+    }
+    #s5 .ui-slider-handle {
+      background: @sky;
+      border-color: @overlay0 !important;
+    }
+    #s6 .ui-slider-handle {
+      background: @sapphire;
+      border-color: @overlay0 !important;
+    }
+    #s7 .ui-slider-handle {
+      background: @blue;
+      border-color: @overlay0 !important;
+    }
+    #s8 .ui-slider-handle {
+      background: @mauve;
+      border-color: @overlay0 !important;
+    }
+    #s9 .ui-slider-handle {
+      background: @pink;
+      border-color: @overlay0 !important;
+    }
```

```suggestion
    .ui-slider-handle {
      box-shadow: inset 0 0 0 4px @base;
      border-color: @overlay0 !important;
    }
    #s0 .ui-slider-handle {
      background: @rosewater;
      /* XXX: do not refactor! an !important in the site's stylesheet won't let this declaration work with a lower specificity */
    }
    #s1 .ui-slider-handle {
      background: @red;
    }
    #s2 .ui-slider-handle {
      background: @peach;
    }
    #s3 .ui-slider-handle {
      background: @yellow;
    }
    #s4 .ui-slider-handle {
      background: @green;
    }
    #s5 .ui-slider-handle {
      background: @sky;
    }
    #s6 .ui-slider-handle {
      background: @sapphire;
    }
    #s7 .ui-slider-handle {
      background: @blue;
    }
    #s8 .ui-slider-handle {
      background: @mauve;
    }
    #s9 .ui-slider-handle {
      background: @pink;
    }
```

Will this not work because of specificity?

---

It will not. There's already an `!important` declaration on the original stylesheet for the `border-color`, so I had to add the `!important`s here.